lib/pages/detail/no_score_message.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:pola_flutter/theme/assets.gen.dart';
+import 'package:pola_flutter/theme/colors.dart';
+import 'package:pola_flutter/theme/fonts.gen.dart';
+import 'package:pola_flutter/theme/text_size.dart';
+import 'package:pola_flutter/i18n/strings.g.dart';
+
+class NoScoreMessage extends StatelessWidget {
+  const NoScoreMessage({Key? key, required this.t}) : super(key: key);
+
+  final Translations t;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 22.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Center(
+            child: Assets.company.unpublished.svg(height: 109.42, width: 109.55),
```

Comment:
To się nie kompiluje. Dodałeś nowy asset, ale musisz odpalić komendę do generowania plików, żeby `Assets.company.unpublished.svg` było dostępne.

Reply:
Musisz przegenerować (i zacommitować) plik `assets.gen.dart`

Reply:
Nadal tego nie zrobiłeś